src/git-repo.ts

```diff
@@ -21,7 +21,7 @@ export class GitRepo {
       if (!existsSync(this.dir.getDirPath())) {
         throw new Error()
       }
-      execSync(`git -C ${this.getDirPath()} status`)
+      execSync(`git -C ${this.getDirPath()} status > /dev/null 2>&1`)
```

Comment:
hi @yeraydavidrodriguez I replaced this line:

```typescript
execSync(`git -C ${this.getDirPath()} status > /dev/null 2>&1`)
```

by:

```typescript
execSync(`git -C ${this.getDirPath()} status`, {stdio: 'ignore'})
```

Reply:
Ok, Is very much expressive that way. Less hackery.